packages/twig/src/patterns/components/tabs/tabs.twig

```diff
@@ -6,7 +6,7 @@
 {% for item in items %}
   {% set tabids = tabids|merge([random(100000)]) %}
 {% endfor %}
-
+<div class="right-to-left">
```

Comment:
This looks like it was done for testing purposes, does this need to be removed?

Reply:
This is to generate id attributes for the tabs since they're needed for aria.

But since we don't know how many tab components will be on a page, we can't assume that naming them after their labels or their index will be sufficiently unique.